domains/foxsden.json

```diff
@@ -4,6 +4,8 @@
         "email": "techseal14@gmail.com"
     },
     "record": {
-        "MX": ["mx1.forwardemail.net"]
+        "MX": "mx1.forwardemail.net",
+        "CNAME": "foxsdenyt.github.io",
```

Comment:
You will need to use A records instead of a CNAME.

Reply:
huh but arent a records for ips?

Reply:
Yes, use the GitHub Pages A records.
```
185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153
```